src/watch_test.ts

```diff
@@ -168,7 +169,11 @@ describe('Watch', () => {
         expect(receivedObjects).to.deep.equal([obj1.object]);
 
         expect(doneCalled).to.equal(true);
-        expect(doneErr).to.deep.equal(errIn);
+        expect(doneErr.length).to.equal(2);
+        if (doneErr.length == 2) {
```

Comment:
This `if check` is redundant as the first check already guards against it.

Reply:
done.